content/4_Gold/Divis.mdx

```diff
@@ -69,6 +77,7 @@ If you've never encountered any **number theory** before, AoPS is a good place t
 	</Resource>
 	<Resource source="CPH" title="21.1 - Primes & Factors" starred></Resource>
 	<Resource source="PAPS" title="16.1, 16.2 - Number Theory"></Resource>
+	<Resource source="MONT@Modern Olympiad Number Theory" title="Chapters 1, 3.1, and 3.2 - Prime Factorization, GCD & LCM, and Divisor Functions" url="https://drive.google.com/file/d/1BcJTLjQaelZ4w_70oHKyImC2I8zLfyrt/view" starred></Resource>
```

Comment:
please keep the title short and in the same format as the others (ex. `1, 3.1, 3.2 - Divisibility`)

Reply:
Ok I'll change it

Reply:
done